src/Symfony/Component/Mime/Crypto/DkimSigner.php

```diff
@@ -30,10 +30,10 @@ final class DkimSigner
     public const ALGO_SHA256 = 'rsa-sha256';
     public const ALGO_ED25519 = 'ed25519-sha256'; // RFC 8463
 
-    private $key;
-    private $domainName;
-    private $selector;
-    private $defaultOptions;
+    private \OpenSSLAsymmetricKey $key;
```

Comment:
Psalm does not yet understand the new return type of `openssl_pkey_get_private()`.

vimeo/psalm#3824